XenAdmin/Wizards/BugToolWizardFiles/BugToolPageDestination.cs

```diff
@@ -48,6 +50,8 @@ namespace XenAdmin.Wizards.BugToolWizardFiles
     public partial class BugToolPageDestination : XenTabPage
     {
         private bool m_buttonNextEnabled;
+        
+       private const int TokenExpiration = 86400; // 24 hours

```

Comment:
I would suggest to have something like TOKEN_EXPIRATION_IN_SECONDS as this is a const value.